ducanhpham0312/zeevision
# feat: Instances page

### Linked issue


Closes ducanhpham0312/zeevision-private#68


### Description

- Add return value when querying instances
- Correct the type of `processId` in `Instance` to `string`
- Implement `InstancesPage`
- Move DeployProcessButton to `ProcessesPage`

### Testing

- Run test-bench `.start.sh` and zeevision `docker compose up --build`
- Run the following query in the playground `localhost:8081/playground`. It should lists all properties of instances defined in the dummy data.
  ```
  {
    instances {
      instanceKey
      processId
      status
      startTime
    }
  }
- Open the application `localhost:8080`. The table of all those instances should be rendered in a table.
